.github/workflows/test_branches.yml

```diff
@@ -542,6 +535,47 @@ jobs:
           #   manage total artifact storage
           # TEST-*.xml
 
+
+  bare-python-env:
+    name: linux/3.6/bare-env
+    runs-on: ubuntu-18.04
+    timeout-minutes: 10
+    steps:
+    - name: Checkout Pyomo source
+      uses: actions/checkout@v2
+
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+
+    - name: Install Pyomo
+      run: |
+        echo ""
+        echo "Install Pyomo..."
+        echo ""
+        python setup.py develop
+        echo ""
+        echo "Set custom PYOMO_CONFIG_DIR"
+        echo ""
+        echo "PYOMO_CONFIG_DIR=${GITHUB_WORKSPACE}/config" >> $GITHUB_ENV
+
+    - name: Run Pyomo standalone test
+      run: |
+        echo ""
+        echo "Running standalone Pyomo test"
+        echo ""
+        python `pwd`/pyomo/environ/tests/standalone_minimal_pyomo_driver.py \
+            || exit 1
+
+    - name: Report pyomo plugin information
```

Comment:
I'd prefer if we "report plugin info" and then run the standalone test.

Reply:
I would, too, I was worried that there might be some lingering side-effect from the first pyomo calls, though: i.e., the initial environment set-up might log additional information to stdout / stderr / loggers, and we wouldn't catch that because it happened during reporting the plugin information.